tools/harness-simulation/posix/sniffer_sim/sniffer.py

```diff
@@ -32,15 +32,24 @@
 import enum
 import grpc
 import logging
+import os
 import signal
-import pcap_codec
+import subprocess
+import tempfile
 import threading
 
+import pcap_codec
 from proto import sniffer_pb2
 from proto import sniffer_pb2_grpc
 import sniffer_transport
 
 
+class CaptureState(enum.Flag):
+    NONE = 0
+    THREAD = enum.auto()
+    ETHERNET = enum.auto()
```

Comment:
Since they'll be used for bit operations, I'd suggest to explicitly specify their values so that code viewers know they're eligible for bit operations.

Reply:
the class `CaptureState` inherits the class `enum.Flag`. I think it is explicitly specified the class will be used for bit operations. And `enum.auto()` in this case will generate the sequence `1, 2, 4, 8, ...`rather than `1, 2, 3, 4, ...`.